{"id":"../../node_modules/core-js/modules/_new-promise-capability.js","dependencies":[{"name":"E:\\2_BACKEND\\NODEJS\\PROJECTS\\natours\\package.json","includedInParent":true,"mtime":1575481576839},{"name":"E:\\2_BACKEND\\NODEJS\\PROJECTS\\natours\\node_modules\\core-js\\package.json","includedInParent":true,"mtime":1574943201579},{"name":"./_a-function","loc":{"line":3,"column":24},"parent":"E:\\2_BACKEND\\NODEJS\\PROJECTS\\natours\\node_modules\\core-js\\modules\\_new-promise-capability.js","resolved":"E:\\2_BACKEND\\NODEJS\\PROJECTS\\natours\\node_modules\\core-js\\modules\\_a-function.js"}],"generated":{"js":"'use strict';\n// 25.4.1.5 NewPromiseCapability(C)\nvar aFunction = require('./_a-function');\n\nfunction PromiseCapability(C) {\n  var resolve, reject;\n  this.promise = new C(function ($$resolve, $$reject) {\n    if (resolve !== undefined || reject !== undefined) throw TypeError('Bad Promise constructor');\n    resolve = $$resolve;\n    reject = $$reject;\n  });\n  this.resolve = aFunction(resolve);\n  this.reject = aFunction(reject);\n}\n\nmodule.exports.f = function (C) {\n  return new PromiseCapability(C);\n};\n"},"sourceMaps":null,"error":null,"hash":"3c6324e0ce1daae835869ec0535da930","cacheData":{"env":{}}}